test-examples/jdi-bdd-no-po-tests/src/test/java/cucmberTests/test/data/User.java

```diff
@@ -0,0 +1,7 @@
+package cucmberTests.test.data;
+
+import com.epam.jdi.tools.DataClass;
+
+public class User extends DataClass<User> {
+    public String name = "Roman", password = "Jdi1234";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5408219)